pkg/gadgets/audit/seccomp/tracer/syscalls_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2025 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSyscallToName(t *testing.T) {
+	tests := []struct {
+		name    string
+		syscall int
+		want    string
+	}{
+		{
+			name:    "syscall 0",
+			syscall: 0,
+			want:    "read",
```

Comment:
This test will not work on all architecture.

https://github.com/seccomp/libseccomp/blob/main/src/syscalls.csv